tests/tom-swifty-test/XmlReflectionTests/SwiftInterfaceParserTests.cs

```diff
@@ -268,6 +268,7 @@ public class Bar : Foo, Nifty {
 		}
 
 		[Test]
+		[Ignore ("Throwing wrong exception ")]
```

Comment:
```suggestion
		[Ignore ("Throwing wrong exception")]
```
Just a little extra space here!